alyx/actions/fixtures/actions.proceduretype.json

```diff
@@ -14,7 +14,7 @@
  "fields": {
   "name": "Water restriction",
   "json": null,
-  "description": "- Variation in the composition, constituents, quantity and/or availability of the diet and/or drinking water which may cause pain, suffering, distress or lasting harm - withholding of water"
+  "description": "- Variation in the compositigit con, constituents, quantity and/or availability of the diet and/or drinking water which may cause pain, suffering, distress or lasting harm - withholding of water"
```

Comment:
I think you've added a typo here ! :lady_beetle:

Reply:
Thanks for catching!

Reply:
Although in the words of Shaggy, '[it wasn't me](https://github.com/cortex-lab/alyx/commit/3f3f50fddd537a6f2f9fef63163f6c07fb34c8d8)'